tekton/assets/monitors/increasing_failed_pipelineruns.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": 2,
+  "created_at": "2024-02-26",
+  "last_updated_at": "2024-02-26",
+  "title": "Increasing number of failed PipelineRuns",
+  "tags": [
+    "integration:tekton"
+  ],
+  "description": "This monitor tracks changes in the number of failed Pipelines in your cluster.",
```

Comment:
Can you add more context here? For reference, see this [guidance](https://docs.datadoghq.com/developers/integrations/create-an-integration-recommended-monitor#description)